library/x509_csr.c

```diff
@@ -78,6 +78,7 @@ static int x509_csr_parse_extensions(mbedtls_x509_csr *csr,
     int ret;
     size_t len;
     unsigned char *end_ext_data;
+    int critical;
```

Comment:
(code style) please leave a blank line between the declared variables and the body of the function

Reply:
Done